frontend/src/pages/my-projects/components/project-card/project-card.tsx

```diff
@@ -13,25 +20,63 @@ type Properties = {
 };
 
 const ProjectCard: React.FC<Properties> = ({ project }: Properties) => {
+  const dispatch = useAppDispatch();
   const { id, name, avatarUrl, category } = project;
 
+  const {
+    isOpenModal: isOpenUpdateProjectModal,
+    handleModalClose: handleUpdateProjectModalClose,
+    handleModalOpen: handleUpdateProjectModalOpen,
+  } = useModal();
+
+  const handleSubmitUpdateProject = useCallback(
+    (payload: ProjectUpdateRequestDto & ProjectUploadImageDto): void => {
+      void dispatch(projectActions.updateProject({ projectId: id, payload }))
+        .unwrap()
+        .then(handleUpdateProjectModalClose);
+    },
+    [dispatch, handleUpdateProjectModalClose, id],
+  );
+
   return (
-    <div className={styles['card']}>
-      <Image className={styles['card-image']} src={avatarUrl ?? img} alt="" />
-      <div className={styles['card-description']}>
-        <Link
-          to={configureString<ValueOf<typeof AppRoute>>(
-            AppRoute.PROJECTS_$PROJECT_ID_SITES,
-            {
-              projectId: id,
-            },
-          )}
-        >
-          <h2>{name}</h2>
-          <p>{category}</p>
-        </Link>
-      </div>
-    </div>
+    <>
+      {isOpenUpdateProjectModal && (
```

Comment:
```suggestion
```
edit modal should be render on the top level

you should use `createPopup` for both. edit and create. it would be better to rename it to the `configurateProject`